reloj_checador_team03/test/system/employees_test.rb

```diff
@@ -0,0 +1,51 @@
+require "application_system_test_case"
+
+class EmployeesTest < ApplicationSystemTestCase
+  setup do
+    @employee = employees(:one)
+  end
+
+  test "visiting the index" do
+    visit employees_url
+    assert_selector "h1", text: "Employees"
+  end
+
+  test "creating a Employee" do
+    visit employees_url
+    click_on "New Employee"
+
+    fill_in "Email", with: @employee.email
+    fill_in "Last name", with: @employee.last_name
+    fill_in "Name", with: @employee.name
+    fill_in "Position", with: @employee.position
+    fill_in "Private number", with: @employee.private_number
+    click_on "Create Employee"
+
+    assert_text "Employee was successfully created"
+    click_on "Back"
+  end
+
+  test "updating a Employee" do
+    visit employees_url
+    click_on "Edit", match: :first
+
+    fill_in "Email", with: @employee.email
+    fill_in "Last name", with: @employee.last_name
+    fill_in "Name", with: @employee.name
+    fill_in "Position", with: @employee.position
+    fill_in "Private number", with: @employee.private_number
+    click_on "Update Employee"
+
+    assert_text "Employee was successfully updated"
+    click_on "Back"
+  end
+
+  test "destroying a Employee" do
+    visit employees_url
+    page.accept_confirm do
+      click_on "Destroy", match: :first
+    end
+
+    assert_text "Employee was successfully destroyed"
+  end
+end
```

Comment:
You are covering so much for a single commit

Considering splitting in more atomic tasks for the next time 😄 
I know are just CRUD implementations, but worth splitting it in order to make it easier to review

Also, saying in the commit-message the "employees feature is done" is not very descriptive.

I think this is something we can improve the next time.

But have to be honest, I'm super impressed with the code-quality in this first commit!

For more info about how to write better commits, read this blog, I'm pretty sure you'd enjoy it: https://chris.beams.io/posts/git-commit/